opengrok-tools/src/main/python/opengrok_tools/scm/repository.py

```diff
@@ -59,6 +61,11 @@ def getCommand(self, cmd, **kwargs):
 
     def sync(self):
         # Eventually, there might be per-repository hooks added here.
+        if isinstance(self.configured_commands, dict) and \
```

Comment:
refactor to a function and use that in incoming() as well ?

Reply:
done